scripts/bluesky-0MQ-proxy

```diff
@@ -9,10 +44,21 @@ if __name__ == "__main__":
                         help='port that RunEngines should broadcast to')
     parser.add_argument('out_port', type=int, nargs=1,
                         help='port that subscribers should subscribe to')
+    parser.add_argument('--verbose', '-v', action='count',
+                        help="Show more log messages. (Use -vvv for even more.)")
+    parser.add_argument('--logfile', type=str,
+                        help="Write logfile ")
```

Comment:
Mention that if it's not set, `--verbose` logs to the screen.

Reply:
When --logfile is not set, `logfile` is `None` which will jump to `else`. `else` is using `print` as default to screen